scripts/examples.js

```diff
@@ -8,8 +8,8 @@ globSync('./examples/*/').forEach((exampleDir) => {
         cwd: exampleDir,
         stdio: 'inherit',
     };
-
-    const result = spawnSync(command, args, opts);
+    let envCommand=command+ (/^win/.test(process.platform) ? '.cmd' : '');
```

Comment:
I think this formatting is a bit confusing. Can you switch it to this (spaces around `=` and `+`:
```js
let envCommand = command + (/^win/.test(process.platform) ? '.cmd' : '');
```